org-code-javabuilder/theater/src/main/java/org/code/SoundLoader.java

```diff
@@ -0,0 +1,17 @@
+package org.code;
+
+public enum Instrument {
+  PIANO, BASS
+}
+
+public static class SoundLoader {
+  /**
+   * Extracts audio samples from WAV or AU file into array
+   * 
+   * @param filename the name of the audio file
+   * @return the array of samples
```

Comment:
I'm not very familiar with how sound is encoded.  Could you elaborate a bit on what the values in this array represent?

Reply:
Each represents a sample value, which you can think of like the position of the speaker at a point in time. Samples played back at a fixed rate (44.1kHz) makes CD-quality sound. This will be taught in the course, I'll add this description to the javadoc